octodns_easydns/__init__.py

```diff
@@ -382,8 +382,7 @@ def _params_for_MX(self, record):
     def _params_for_SRV(self, record):
         for value in record.values:
             yield {
-                'rdata': f"{value.priority} {value.port} {value.weight} "
-                f"{value.target}",
+                'rdata': f'-{value.priority} {value.weight} {value.port} {value.target}',
```

Comment:
This appears to be making the priority negative with a `-` on the front. Is that intentional?

Reply:
You are correct. I had set an extra value when trying to figure out why no changes were taking effect (i was in the wrong environment), and forgot to fix it. The commit below (48d8434) removes that.